### PR TITLE
Test/login redirect

### PR DESCRIPTION
**Pull request by LBHTKarki**
# What?
1. fix an issue where missing cookie caused infinite loop in the login redirect process
2. make ALLOWED_GROUPS env var global in serverless config so it's available to all functions. Required by isLoggedIn.ts for example
3. Update serverless config to resolve an issue where the root of the application failed to load with "missing authentication token" error

# Why?
To resolve some minor issues with the login process and in serverless config

### CODE REVIEW

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -19,17 +19,22 @@ MaTApp.getInitialProps = async (context: any) => {
...
     ) {
       const appProps = await App.getInitialProps(context);
       return { ...appProps }
     }
   }
 
-  context.ctx.res.writeHead(302, { Location: unauthenticatedLandingPage });
-  context.ctx.res.end();
-  return {};
+  if(context.ctx.pathname !== '/login-redirect')
```
Comment (mackella) [inline src/pages/_app.tsx:29]: Can we use the `unauthenticatedLandingPage` constant here please?

File: src/pages/_app.tsx [hunk 2]
```diff
@@ -19,17 +19,22 @@ MaTApp.getInitialProps = async (context: any) => {
...
+  {
+    context.ctx.res.writeHead(302, { Location: unauthenticatedLandingPage });
+    context.ctx.res.end();
+    return {};
+  }
+  else{
+    return{};
```
Comment (mackella) [inline src/pages/_app.tsx:36]: This should really evaluate and return the downstream props, as per lines 24/25.

File: src/pages/_app.tsx (hunk 1 above)
Comment (LBHTKarki) [reply src/pages/_app.tsx]: sorry my mistake. I've pushed a fix.

File: src/pages/_app.tsx (hunk 2 above)
Comment (LBHTKarki) [reply src/pages/_app.tsx]: I've pushed an update for this

**Pull request merged**